src/io/flutter/actions/FlutterSdkAction.java

```diff
@@ -30,19 +32,43 @@
   public void actionPerformed(AnActionEvent event) {
     final Project project = DumbAwareAction.getEventProject(event);
 
-    final FlutterSdk sdk = project != null ? FlutterSdk.getFlutterSdk(project) : null;
-    if (sdk == null) {
-      showMissingSdkDialog(project);
-      return;
+    // See if the Bazel workspace exists for this project.
+    final Workspace workspace = project != null ? WorkspaceCache.getInstance(project).getNow() : null;
```

Comment:
Nit: don't we only need to access the workspace if `enableActionInBazelContext` returns true?